.github/workflows/android-check.yaml

```diff
@@ -131,6 +131,15 @@ jobs:
           ninja --version
           ./gradlew -P${{ matrix.arch }} assemble${{ matrix.flavor }}
 
+      - name: Free disk space for emulator
+        if: matrix.run-tests == true
+        uses: AdityaGarg8/remove-unwanted-software@v5
```

Comment:
Do we really need a 3party action for a few rm -rf commands from https://github.com/AdityaGarg8/remove-unwanted-software/blob/master/action.yml ?

Reply:
why not?

Reply:
If someone hacks this action, anything bad may happen, like https://unit42.paloaltonetworks.com/github-actions-supply-chain-attack/